src/test/java/com/ipiecoles/java/java350/model/EmployeTest.java

```diff
@@ -66,4 +96,213 @@ public void testNombreAnneeAncienneteNull(){
         Assertions.assertThat(nbAnneesAnciennete).isEqualTo(0);
     }
 
+
+    
+    //EVAL---------------------------------------------------
+//----------------------------------------------------- augmenterSalaire--------------------------------------------------------------------//
+//Test salaire si fonctionne
+    @Test
+    public void testAugmenterSalaire(){
+        //Given
+        Double pourc = 1.3; //Augmentation de 1.3 donc 30 pourcents
+        Employe employe = new Employe();
+        employe.setSalaire(1000d);
+
+        //When
+        employe.augmenterSalaire(pourc);
+
+        //Then
+        Assertions.assertThat(employe.getSalaire()).isGreaterThan(1000d);
+
+    }
+
+    //Test si le pourcentage est négatif
+    @Test
+    public void testAugmenterSalaireNegatif(){
+        //Given
+        Double pourc = -10d;
+        Employe employe = new Employe();
+        employe.setSalaire(1000d);
+        Double res = employe.getSalaire();
+
+        //When
+        employe.augmenterSalaire(pourc);
+
+        //Then
+        Assertions.assertThat(employe.getSalaire()).isEqualTo(res);
+    }
+
+    //Test salaire si pourcentage infèrieur à 1
+    @Test
+    public void testAugmenterSalaireInfUn(){
+        //Given
+        Double pourc = 0d;
+        Employe employe = new Employe();
+        employe.setSalaire(1000d);
+        Double res = employe.getSalaire();
+
+        //When
+        employe.augmenterSalaire(pourc);
+
+        //Then
+        Assertions.assertThat(employe.getSalaire()).isEqualTo(res);
+
+    }
+
+
+//si l'employé n'a pas encore de salaire inscrit.
+    @Test
+    public void testAugmenterSalairePasSalaire(){
+        //Given
+        Double pourc = 1.3d;
+        Employe employe = new Employe();
+        employe.setSalaire(null);
+
+
+        //hasMessage permet de vérifier si augmenterSalaire à Trhow que le salaire était null
+        Assertions.assertThatThrownBy(() -> employe.augmenterSalaire(pourc)).hasMessage("une erreur est survenue le salaire est null");
+
+    }
+
+    //Test paramétré pour la NbRTT
+    @ParameterizedTest
+    @CsvSource({
+            "'2019-01-01', 8",//(365 - 218 - 104 - 10 - 25 = 8)*1 car travail 100 pourcent du temps
+            "'2021-01-01', 10",
+            "'2022-01-01', 10",//(365 - 218 - 105 - 7 - 25 = 10)*1     105 samedi et dimanche
+            "'2032-01-01', 11" })
+    public void testNbRtt(LocalDate date, Integer nbRtt) {
+        Employe employe = new Employe("Saint", "Exupery", "M00123", LocalDate.now(), 1600.0, 1, 1.0);
```

Comment:
Il doit te manquer un peu de couverture ici, les jeudis (à tester avec une année 2026) et vendredis (année 2016)en année bissextilles
pour faire celui du vendredi il faut ajouter  
datePaque.put(2016, LocalDate.of(2016, 3, 27)); // Ajout pour le test 2016 de getNbRtt
dans entreprise.java

Reply:
Oui la couverture pourrait être améliorée mais le test est bien. Pense aux //Given //When //Then